learn/advanced/storage.md

```diff
@@ -1,20 +1,20 @@
 # Storage
 
-::: tip
+Meilisearch is, in many ways, a database. It stores indexed documents along with the data needed to return relevant search results.
```

Comment:
Hmmm…that's a good question. I added "in many ways" because while Meilisearch is a database, we also do a lot of things which go beyond that, no? You wouldn't use Meilisearch the same way you'd use MySQL, would you?

Reply:
Well technically you can. But MySQL is one type of database with some use cases, you also have other types of database with other uses.

Here is a very nice video about the different types of databases (well most of) https://www.youtube.com/watch?v=W2Z7fbCLSTw